packages/grafana-prometheus/src/datasource.ts

```diff
@@ -71,6 +73,13 @@ const GET_AND_POST_METADATA_ENDPOINTS = ['api/v1/query', 'api/v1/query_range', '
 
 export const InstantQueryRefIdIndex = '-Instant';
 
+const filterOperatorMap: Record<string, ScopeFilterOperator> = {
+  '=': 'equals',
+  '!=': 'not-equals',
+  '=~': 'regex-match',
+  '!~': 'regex-not-match',
+};
+
```

Comment:
Maybe this should be in `scopes.ts`. I am not sure about this.

Reply:
Yeah, I would move this to `scopes.ts`.